packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -137,8 +155,26 @@ export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
         cursor: 'pointer',
         flexBasis: '32px',
         flexShrink: 0,
-        padding: 1,
-        color: palette.themePrimary
+        padding: 0,
+        margin: '-1px 0px',
+        selectors: {
+          '&:hover .ms-Button': {
+            backgroundColor: inputBackgroundHovered
+          },
+          '&:hover .ms-Button-icon': {
+            color: inputIconAltHovered
+          },
+          '.ms-Button': getRTL()
```

Comment:
This whole thing can be simplified like this. It’s much more readable.
```ts
‘.ms-Button’: {
  borderRadius: getRTL() ? ‘1px 0 0 1px’ : ‘0 1px 1px 0’
}
```

Reply:
Not at all, thanks so much for taking the time to review so thoroughly! I'll get these changes in.